.github/workflows/check-advisories.yml

```diff
@@ -0,0 +1,30 @@
+name: Check security advisories
+on:
+  workflow_call:
+    inputs:
+      artifact-name:
+        required: true
+        type: string
+jobs:
+  check-advisories:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: source
+      - run: mkdir -p ~/.local/bin
+      - id: download
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.artifact-name }}
+          path: ~/.local/bin
+      - run: chmod +x ~/.local/bin/hsec-tools
+      - name: run checks
+        run: |
+          cd source
+          RESULT=0
+          for FILE in $(find advisories EXAMPLE_ADVISORY.md -type f -name "*.md") ; do
```

Comment:
Do we intend to limit advisories names? [see for vs while](https://unix.stackexchange.com/a/24278/373072)


```suggestion
          ls advisories/*.md EXAMPLE_ADVISORY.md | while read -r FILE ; do
```

Reply:
@blackheaven thanks for pointing that out.  I've modified it, but still using `find` so that we can handle a nested structure (if we want that).

Reply:
I think we should enforce a particular structure, otherwise we might check eventual files associated with a particular advisory.

Reply:
I think it's a bit early to predict what we will need, or how that could change over time.  If you're satisfied with the current behaviour, we can merge it.  We will refine or evolve it as needed.